crates/bitwarden-ipc/src/traits/communication_backend.rs

```diff
@@ -1,14 +1,22 @@
+use std::fmt::Debug;
+
 use crate::message::{IncomingMessage, OutgoingMessage};
 
 /// This trait defines the interface that will be used to send and receive messages over IPC.
 /// It is up to the platform to implement this trait and any necessary thread synchronization and
 /// broadcasting.
-pub trait CommunicationBackend {
-    type SendError;
+pub trait CommunicationBackend: Send + Sync + 'static {
+    type SendError: Debug + Send + Sync + 'static;
```

Comment:
Non-blocking thought: We could also have the Errors require the Error trait, which requires Debug on its own, though this might require creating a new error type for the tests, so maybe not worth it.
```suggestion
    type SendError: std::error::Error + Send + Sync + 'static;
```

Reply:
Not using `std::error` allow us to use an arbitrary `JsValue` from a captured exception and bubble that up. Though of course, now that we are going across thread boundaries that is no longer possible because of other reasons, but still, maybe some other platform can take advantage of it?

Reply:
Yeah that's reasonable enough, we can always add the bound later if it turns out we only use Errors.